lib/sqlalchemy/dialects/mysql/aiomysql.py

```diff
@@ -255,16 +256,17 @@ def _init_dbapi_attributes(self):
 
     def connect(self, *arg, **kw):
         async_fallback = kw.pop("async_fallback", False)
+        creator_fn = kw.pop("creator_fn", partial(self.aiomysql.connect))
```

Comment:
There is no need to use partial here or in the other cases

Reply:
Removed use of `partial` from all dialects